File: Module Four/User-Defined Functions/pyramidVolume.cpp
#include <iostream>
using namespace std;
   
   double PyramidVolume(double baseLength, double baseWidth, double pyramidHeight){
	double baseArea = baseLength * baseWidth;
	double vol = ((baseArea * pyramidHeight) * 1/3);
	return vol;
}
   

int main() {
   double userLength;
   double userWidth;
   double userHeight;

   cin >> userLength;
   cin >> userWidth;
   cin >> userHeight;

   cout << "Volume: " << PyramidVolume(userLength, userWidth, userHeight) << endl;

   return 0;
}

File: Module Ten/Python: List Basics/pyramid_volume.py

def pyramid_volume(base_length, base_width, pyramid_height):
    return(base_width * base_length * pyramid_height) / 3

length = float(input())
width = float(input())
height = float(input())
print('Volume for 4.5, 2.1, 3.0 is:', pyramid_volume(length, width, height))
